workers/data_refinery_workers/processors/qn_reference.R

```diff
@@ -0,0 +1,59 @@
+###
+# qn_reference
+# Alex's Lemonad Stand Foundation
+# Childhood Cancer Data Lab
+#
+###
+
+#######################
+# The command interface!
+#######################
+
+library("optparse")
+library(data.table)
+library("preprocessCore")
+
+option_list = list(
+  make_option(c("-i", "--inputFile"), type="character", default="", 
+              help="inputFile", metavar="character"),
+  make_option(c("-o", "--outputFile"), type="character", default="", 
+              help="outputFile", metavar="character")
+); 
+
+opt_parser = OptionParser(option_list=option_list);
+opt = parse_args(opt_parser);
+
+filePath <- opt$inputFile
+outFilePath <- opt$outputFile
+
+# Get and use this DB
```

Comment:
I don't think we need lines 29-31 since we won't be using any annotation `.db` packages or installing bioconductor packages.